usage/jsgui/src/main/webapp/assets/js/config.js

```diff
@@ -48,6 +50,15 @@ require.config({
         "zeroclipboard":"libs/ZeroClipboard",
         "js-yaml":"libs/js-yaml",
         
+        "codemirror":"lib/codemirror",
```

Comment:
From: https://codemirror.net/doc/manual.html#modloader
`Do not use RequireJS' paths option to configure the path...`